version/version_test.go

```diff
@@ -16,31 +16,54 @@ limitations under the License.
 package version
 
 import (
+	"fmt"
+	"os"
 	"path/filepath"
 	"runtime"
 	"testing"
 
 	mf "github.com/manifestival/manifestival"
+	"knative.dev/operator/pkg/reconciler/common"
 )
 
 func TestManifestVersionServingSame(t *testing.T) {
```

Comment:
We should get rid of this file entirely.

Reply:
Done.